contrib/build_executable.jl

```diff
@@ -123,47 +127,48 @@ function build_executable(exename, script_file, targetdir=nothing, cpu_target="n
         end
     end
 
-    for exe in [exe_release, exe_debug]
-        println("running: $gcc -g -Wl,--no-as-needed $win_arg $(join(incs, " ")) $(cfile) -o $(exe.buildfile) -Wl,-rpath,$(sys.buildpath) -L$(sys.buildpath) $(exe.libjulia) -l$(exename)")
-        cmd = setenv(`$gcc -g -Wl,--no-as-needed $win_arg $(incs) $(cfile) -o $(exe.buildfile) -Wl,-rpath,$(sys.buildpath) -L$(sys.buildpath) $(exe.libjulia) -l$(exename)`, ENV2)
-        run(cmd)
-        println()
-    end
+    linker_rpath = portable ? `` : `-Wl,-rpath,$(sys.buildpath)`
+    println("running: $gcc -g -Wl,--no-as-needed $win_arg $(join(incs, " ")) $(cfile) -o $(exe_file.buildfile) $(linker_rpath) -L$(sys.buildpath) $(exe_file.libjulia) -l$(exename)")
+    cmd = setenv(`$gcc -g -Wl,--no-as-needed $win_arg $(incs) $(cfile) -o $(exe_file.buildfile) $(linker_rpath) -L$(sys.buildpath) $(exe_file.libjulia) -l$(exename)`, ENV2)
+    run(cmd)
+    println()
 
-    println("running: rm -rf $(tmpdir) $(sys.buildfile).o $(sys.buildfile0).o $(sys.buildfile0).ji")
-    map(f-> rm(f, recursive=true), [tmpdir, sys.buildfile*".o", sys.buildfile0*".o", sys.buildfile0*".ji"])
+    println("running: rm -rf $(tmpdir) $(sys.buildfile * ".o") $(sys.buildfile0 * ".o") $(sys.buildfile0 * ".ji")")
+    run(`rm -rf $(tmpdir) $(sys.buildfile * ".o") $(sys.buildfile0 * ".o") $(sys.buildfile0 * ".ji")`)
```

Comment:
use the julia `rm` function, not the coreutils `rm` which is not going to be present on windows